### ISSUE TITLE
:checkhealth warns about not finding lua on macOS with WARNING lua: not found

### ISSUE THREAD

**Issue by craigmac** [nvim-neorocks/rocks.nvim#408]
On macOS the `brew install lua` is 5.4, which is not what we want, and `brew install lua@5.1` has [been disabled](https://github.com/Homebrew/homebrew-core/commit/57d73df3ba924c870f4b6d36080e7cb37275367e). An alternative is to just install `brew install luajit`, but it seems rocks.nvim is still looking to find `lua` binary, and so warns it can't find lua with this output in `:checkhealth`:

```
rocks: require("rocks.health").check()

Checking external dependencies ~
- OK luarocks: found 
- WARNING         lua: not found.
          Install [Lua](https://www.lua.org/) for extended capabilities.
          luarocks requires a Lua installation.
```

Error when trying to run `:Rocks sync`, probably due to no `lua` binary on the system (it should try `luajit`also?):

```
Error executing vim.schedule lua callback: ...hern/.local/share/nvim/rocks/share
/lua/5.1/nio/tasks.lua:100: Async task failed without callback: The coroutine fa
iled with this message:                                                         
...rn/.local/share/nvim/rocks/share/lua/5.1/nio/control.lua:145: env: lua: No su
ch file or directory
[...snip...]
```

**Comment by mrcjkb** [comment]
Hey 👋 

Thanks for reporting 🙏 

Related: https://github.com/nvim-neorocks/rocks.nvim/discussions/385

This is actually a Homebrew issue (it should provide a `lua` symlink to `luajit`).

We might be able to solve this with https://github.com/nvim-neorocks/rocks.nvim/issues/386, which (if it works) would get luarocks to use neovim as the Lua interpreter.

**Comment by craigmac** [comment]
Thanks for pointing out the related issue, I've responded on the discussion with [what I've done](https://github.com/nvim-neorocks/rocks.nvim/discussions/385#discussioncomment-9863470) to fix this on macOS (installed lua 5.1 from source download). I'll close this.
